extensions/amp-bind/0.1/test/test-bind-expression.js

```diff
@@ -69,6 +69,9 @@ describe('BindExpression', () => {
   it('should evaluate ternary operator', () => {
     expect(evaluate('true ? "a" : "b"')).to.be.equal('a');
     expect(evaluate('false ? "a" : "b"')).to.be.equal('b');
+    expect(evaluate('true ?: "a"')).to.be.true;
+    expect(evaluate('"hello" ? : "a"')).to.be.equal('hello');
```

Comment:
Do you foresee any issues if we continue to allow the space? [Other languages allow a space here](https://en.wikipedia.org/wiki/%3F%3A#C). Developers familiar with the operator may wish to use it that way.